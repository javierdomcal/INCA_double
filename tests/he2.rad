      0.000000000000E+00       0.000000000000E+00
      0.100000000000E+00       0.467497135410E-01
      0.200000000000E+00       0.175955646171E+00
      0.300000000000E+00       0.360655654773E+00
      0.400000000000E+00       0.569110707935E+00
      0.500000000000E+00       0.772923575702E+00
      0.600000000000E+00       0.951421352005E+00
      0.700000000000E+00       0.109252245053E+01
      0.800000000000E+00       0.119121822997E+01
      0.900000000000E+00       0.124750149863E+01
      0.100000000000E+01       0.126467188832E+01
      0.110000000000E+01       0.124810009163E+01
      0.120000000000E+01       0.120428162157E+01
      0.130000000000E+01       0.114005505871E+01
      0.140000000000E+01       0.106196169948E+01
      0.150000000000E+01       0.975779094287E+00
      0.160000000000E+01       0.886254961874E+00
      0.170000000000E+01       0.797030942643E+00
      0.180000000000E+01       0.710711580484E+00
      0.190000000000E+01       0.629019609552E+00
      0.200000000000E+01       0.552983682243E+00
      0.210000000000E+01       0.483120288157E+00
      0.220000000000E+01       0.419588818843E+00
      0.230000000000E+01       0.362312329738E+00
      0.240000000000E+01       0.311065116549E+00
      0.250000000000E+01       0.265532492425E+00
      0.260000000000E+01       0.225349507777E+00
      0.270000000000E+01       0.190125033975E+00
      0.280000000000E+01       0.159456507194E+00
      0.290000000000E+01       0.132939246863E+00
      0.300000000000E+01       0.110172944496E+00
      0.310000000000E+01       0.907668300170E-01
      0.320000000000E+01       0.743442265629E-01
      0.330000000000E+01       0.605466943151E-01
      0.340000000000E+01       0.490376917936E-01
      0.350000000000E+01       0.395055851020E-01
      0.360000000000E+01       0.316658479835E-01
      0.370000000000E+01       0.252623647219E-01
      0.380000000000E+01       0.200678347169E-01
      0.390000000000E+01       0.158833570758E-01
      0.400000000000E+01       0.125373328014E-01
      0.410000000000E+01       0.988385693172E-02
      0.420000000000E+01       0.780078484417E-02
      0.430000000000E+01       0.618765051909E-02
      0.440000000000E+01       0.496359569143E-02
      0.450000000000E+01       0.406544314992E-02
      0.460000000000E+01       0.344601967582E-02
      0.470000000000E+01       0.307280768587E-02
      0.480000000000E+01       0.292698170870E-02
      0.490000000000E+01       0.300286745318E-02
      0.500000000000E+01       0.330784772350E-02
      0.500000000000E+01       0.330784772350E-02
      0.505000000000E+01       0.355220669304E-02
      0.510000000000E+01       0.386273068795E-02
      0.515000000000E+01       0.424425654070E-02
      0.520000000000E+01       0.470259184479E-02
      0.525000000000E+01       0.524456804195E-02
      0.530000000000E+01       0.587810152622E-02
      0.535000000000E+01       0.661226301597E-02
      0.540000000000E+01       0.745735552357E-02
      0.545000000000E+01       0.842500135028E-02
      0.550000000000E+01       0.952823865535E-02
      0.555000000000E+01       0.107816282946E-01
      0.560000000000E+01       0.122013717991E-01
      0.565000000000E+01       0.138054415704E-01
      0.570000000000E+01       0.156137246072E-01
      0.575000000000E+01       0.176481813510E-01
      0.580000000000E+01       0.199330215436E-01
      0.585000000000E+01       0.224948993242E-01
      0.590000000000E+01       0.253631301585E-01
      0.595000000000E+01       0.285699325684E-01
      0.600000000000E+01       0.321506980189E-01
      0.605000000000E+01       0.361442926904E-01
      0.610000000000E+01       0.405933952028E-01
      0.615000000000E+01       0.455448746300E-01
      0.620000000000E+01       0.510502133051E-01
      0.625000000000E+01       0.571659789231E-01
      0.630000000000E+01       0.639543502269E-01
      0.635000000000E+01       0.714837000469E-01
      0.640000000000E+01       0.798292385505E-01
      0.645000000000E+01       0.890737181605E-01
      0.650000000000E+01       0.993081996060E-01
      0.655000000000E+01       0.110632875884E+00
      0.660000000000E+01       0.123157947468E+00
      0.665000000000E+01       0.137004537892E+00
      0.670000000000E+01       0.152305633905E+00
      0.675000000000E+01       0.169207028999E+00
      0.680000000000E+01       0.187868243499E+00
      0.685000000000E+01       0.208463389248E+00
      0.690000000000E+01       0.231181942730E+00
      0.695000000000E+01       0.256229388263E+00
      0.700000000000E+01       0.283827693166E+00
      0.705000000000E+01       0.314215580292E+00
      0.710000000000E+01       0.347648569706E+00
      0.715000000000E+01       0.384398769630E+00
      0.720000000000E+01       0.424754404372E+00
      0.725000000000E+01       0.469019069406E+00
      0.730000000000E+01       0.517510694351E+00
      0.735000000000E+01       0.570560164191E+00
      0.740000000000E+01       0.628509486868E+00
      0.745000000000E+01       0.691709289341E+00
      0.750000000000E+01       0.760515263834E+00
      0.755000000000E+01       0.835282964928E+00
      0.760000000000E+01       0.916360078668E+00
      0.765000000000E+01       0.100407496295E+01
      0.770000000000E+01       0.109871992949E+01
      0.775000000000E+01       0.120052746630E+01
      0.780000000000E+01       0.130963749996E+01
      0.785000000000E+01       0.142605407166E+01
      0.790000000000E+01       0.154959078816E+01
      0.795000000000E+01       0.167980654463E+01
      0.800000000000E+01       0.181593650782E+01
      0.805000000000E+01       0.195682733069E+01
      0.810000000000E+01       0.210088707751E+01
      0.815000000000E+01       0.224605473231E+01
      0.820000000000E+01       0.238977965215E+01
      0.825000000000E+01       0.252898743696E+01
      0.830000000000E+01       0.266001982527E+01
      0.835000000000E+01       0.277859444870E+01
      0.840000000000E+01       0.287991543650E+01
      0.845000000000E+01       0.295909267384E+01
      0.850000000000E+01       0.301190204157E+01
      0.855000000000E+01       0.303566577014E+01
      0.860000000000E+01       0.302984729683E+01
      0.865000000000E+01       0.299604703537E+01
      0.870000000000E+01       0.293742684692E+01
      0.875000000000E+01       0.285790321203E+01
      0.880000000000E+01       0.276148656129E+01
      0.885000000000E+01       0.265193582465E+01
      0.890000000000E+01       0.253266569164E+01
      0.895000000000E+01       0.240675477074E+01
      0.900000000000E+01       0.227694896188E+01
      0.905000000000E+01       0.214563703584E+01
      0.910000000000E+01       0.201482200898E+01
      0.915000000000E+01       0.188611380748E+01
      0.920000000000E+01       0.176075112009E+01
      0.925000000000E+01       0.163964554209E+01
      0.930000000000E+01       0.152343608289E+01
      0.935000000000E+01       0.141254394578E+01
      0.940000000000E+01       0.130722152130E+01
      0.945000000000E+01       0.120759304211E+01
      0.950000000000E+01       0.111368659482E+01
      0.955000000000E+01       0.102545837081E+01
      0.960000000000E+01       0.942810540398E+00
      0.965000000000E+01       0.865604238957E+00
      0.970000000000E+01       0.793669034210E+00
      0.975000000000E+01       0.726810012198E+00
      0.980000000000E+01       0.664813346093E+00
      0.985000000000E+01       0.607450946913E+00
      0.990000000000E+01       0.554484567519E+00
      0.995000000000E+01       0.505669555055E+00
      0.100000000000E+02       0.460758324367E+00
      0.100500000000E+02       0.419503550340E+00
      0.101000000000E+02       0.381661041146E+00
      0.101500000000E+02       0.346992246147E+00
      0.102000000000E+02       0.315266361649E+00
      0.102500000000E+02       0.286262016122E+00
      0.103000000000E+02       0.259768537531E+00
      0.103500000000E+02       0.235586824617E+00
      0.104000000000E+02       0.213529859025E+00
      0.104500000000E+02       0.193422904983E+00
      0.105000000000E+02       0.175103448107E+00
      0.105500000000E+02       0.158420925470E+00
      0.106000000000E+02       0.143236296405E+00
      0.106500000000E+02       0.129421498748E+00
      0.107000000000E+02       0.116858829230E+00
      0.107500000000E+02       0.105440280330E+00
      0.108000000000E+02       0.950668596499E-01
      0.108500000000E+02       0.856479120660E-01
      0.109000000000E+02       0.771004598065E-01
      0.109500000000E+02       0.693485712493E-01
      0.110000000000E+02       0.623227656193E-01
      0.110500000000E+02       0.559594578610E-01
      0.111000000000E+02       0.502004456790E-01
      0.111500000000E+02       0.449924390012E-01
      0.112000000000E+02       0.402866308351E-01
      0.112500000000E+02       0.360383075835E-01
      0.113000000000E+02       0.322064962892E-01
      0.113500000000E+02       0.287536459283E-01
      0.114000000000E+02       0.256453397083E-01
      0.114500000000E+02       0.228500353179E-01
      0.115000000000E+02       0.203388301647E-01
      0.115500000000E+02       0.180852488077E-01
      0.116000000000E+02       0.160650500087E-01
      0.116500000000E+02       0.142560510703E-01
      0.117000000000E+02       0.126379673861E-01
      0.117500000000E+02       0.111922653808E-01
      0.118000000000E+02       0.990202726170E-02
      0.118500000000E+02       0.875182623019E-02
      0.119000000000E+02       0.772761100419E-02
      0.119500000000E+02       0.681659868502E-02
      0.120000000000E+02       0.600717515899E-02
