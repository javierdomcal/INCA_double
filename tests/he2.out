 *******Job Finished************
 -----------Computational time----------------
 Total CPU time   2.84948707    
 Grid points computing time   8.47689956E-02
 Primitive quartet loop time   2.76384115    
 CPU time for intracule integrations   8.76903534E-04
 ---Primitive quartet time analysis-----------
 Reading .dm2p file-->   2.92242318E-03
 1st primitive screening-->  -5.07596135E-03
 2nd primitive screening-->   1.56135857E-03
 Grid points loop-->   2.57432127    
 ---------------------------------------------
 ------Grid point + primitive quartet info----
 Original grid points           0
 Symmetry reduced grid points       11761
 Total number of reduced grid points           0
 ---------------------------------------------
 --------------Accuracy check-----------------
 Sum of all DM2prim terms=   14.568082367977475        17.352773362142422     
 Thresholds used for DM2prim   10.000000000000000        10.000000000000000     
 Threshold used for screenings(Tau)=   1.0000000000000000E-008
 Computed limit for the screenings   2.7777777777777777E-010
 ---------------------------------------------
 RADIAL PLOT, I(S) vs s
 Number of distances         192
 From   0.0000000000000000      to   12.000000000000000     
